docusaurus.config.js

```diff
@@ -3452,7 +3452,25 @@ module.exports = {
           {
             to: '/v2.7/reference-guides/best-practices/rancher-server/tuning-and-best-practices-for-rancher-at-scale',
             from: '/v2.7/reference-guides/best-practices/rancher-server/tips-for-scaling-rancher'
+          },
+          // Redirects for restructure from PR #1147 (start)
+          {
+            to: '/v2.8/api/v3-rancher-api-guide',
+            from: '/v2.8/reference-guides/about-the-api'
```

Comment:
The from field can take an array so that we can avoid multiple entries for the same redirect target. The block from 1720-1723 can be removed and combined with this one.

```suggestion
            from: ['/v2.8/reference-guides/about-the-api', '/v2.8/pages-for-subheaders/about-the-api']
```

Reply:
Just updated with removal of lines 1720-1723, 2064-2067, and consolidated the redirect links. Thanks for the review Billy!